examples/deepspeed/cifar10_cpu_offloading/README.md

```diff
@@ -0,0 +1,92 @@
+# DeepSpeed CPU Offloading example
+This example is adapted from the 
+[CIFAR example in the DeepSpeedExamples](https://github.com/microsoft/DeepSpeedExamples/tree/master/cifar) 
+repository. It is intended to show how to configure 
+[ZeRO Stage 3 with CPU offloading](https://www.deepspeed.ai/tutorials/zero/) for a simple CNN network.
+
+Compared to the original example, MoE is removed due to `AssertionError: MoE not supported with Stage 3`.
+
+**IMPORTANT!** 
+ZeRO Stage 3 allows to train model exceeding GPU memory by offloading optimizer, parameters and gradients to CPU
```

Comment:
suggested edit: "ZeRO Stage 3 enables training models that do not fit into GPU memory by..."